### PR TITLE
Extract the common part to calculate the layout for Flexbox as FlexboxHelper

### PR DESCRIPTION
**Pull request by thagikura**
This PR only extracts the code to reorder the indices. The rest of the calculation will be also extracted in the future PRs.

### CODE REVIEW

**Comment by thagikura** [comment]
Could you take a look @yaraki ?

File: flexbox/src/main/java/com/google/android/flexbox/FlexboxHelper.java [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * Offers various calculations for Flexbox to use the common logic between the classes such as
+ * {@link FlexboxLayout} and {@link FlexboxLayoutManager}.
+ */
+public class FlexboxHelper {
```
Comment (yaraki) [inline flexbox/src/main/java/com/google/android/flexbox/FlexboxHelper.java]: Can't this be package-private?

Comment (thagikura) [inline flexbox/src/main/java/com/google/android/flexbox/FlexboxHelper.java]: Good point. Done.

**Comment by thagikura** [comment]
@yaraki Thanks. Addressed. I unintentionally amended the last commit. But the change from the last review was removing the `public` qualifier from the FlexHelper. I hope it doesn't make the reivew hard.

**Comment by yaraki** [comment]
+2!

**Comment by thagikura** [comment]
Thanks! Merged.

**Pull request merged**